src/content/docs/develop/resources.mdx

```diff
@@ -34,12 +35,22 @@ Alternatively the `resources` config also accepts a map object if you want to ch
     "resources": {
       "/absolute/path/to/textfile.txt": "resources/textfile.txt",
       "relative/path/to/jsonfile.json": "resources/jsonfile.json",
-      "resources/*": "resources/"
+      "resources/*": "resources/",
+      "resources/**/*": "resources/"
     }
   }
 }
 ```
 
+The syntax for resource source paths works like this:
+
+- `"dir/file.txt"`: Will copy the `file.txt` file into the target resource directory.
+- `"dir/*"` / `dir/**`: Will copy all files in the `dir` directory _non-recursively_ (sub-directories will be ignored) into the target resource directory.
+- `"dir/**/*"`: Will copy all files in the `dir` directory _recursively_ (all files in all sub-directories) into the target resource directory. Note, this does **not** include the files from `dir/*`, but only those in sub-directories.
+- `"dir/"`: Will copy all files **and directories** _recursively_ into the target resource directory. Use this if you also want to preserve the file system structure of your files and directories.
+
+The `target resource directory` is either the right value of the object notation, or a reconstruction of the source path in the array notation.
```

Comment:
This was a bit confusing at first and I had to read it multiple times to know what it meant, maybe using `"` instead of a backtick is better here. A better solution would be to add intralinks where "target resource directory" is mentioned.

I also think this section should have its own header